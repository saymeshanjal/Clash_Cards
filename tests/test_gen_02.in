10
10 5 3
4 5 4
14 24 2
4 1 2
8 15 5
2 4 3
2 36 3
2 45 4
12 42 4
14 9 1