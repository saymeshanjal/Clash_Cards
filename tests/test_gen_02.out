6 246 0
0 6 0
77 378 2977
0 3 3
0 70 0
0 2 0
0 2 0
0 2 0
77 477 370
1558 1874 0
